src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -6882,6 +7294,40 @@ private void SaveError(IScriptExtent extent, Expression<Func<string>> errorExpr,
             SaveError(errorToSave);
         }
 
+        /// <summary>
+        /// Assertion to ensure that .resx resource files are where all errors
+        /// saved by the parser are stored. Goes through the known resource types
```

Comment:
Could you please make the comment more simple?

Reply:
I'd prefer to document why this function exists, since it does something a bit odd and annotations will mean it only exists in Debug builds. Is there a way you'd like me to simplify it? I can just imagine someone coming across this function in a year or two and having to go through all the resource file stuff to work out what's going on

Reply:
I means that it is very complex English sentence `... that .. are where ... by ... are stored`.

Reply:
Yes, after rereading I agree. Does the new wording work better?